src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -60,6 +65,7 @@ private static ArrayList<CommandMetadata> initMetadataList() {
         list.add(new CommandMetadata(REMOVE_MODULE_PATTERN, REMOVE_MODULE_ARGUMENTS, REMOVE_MODULE_CONSTRUCTOR));
         list.add(new CommandMetadata(ADD_MODULE_PATTERN, ADD_MODULE_ARGUMENTS, ADD_MODULE_CONSTRUCTOR));
         list.add(new CommandMetadata(GRADE_PATTERN, GRADE_ARGUMENTS, GRADE_CONSTRUCTOR));
+        list.add(new CommandMetadata(BYE_PATTERN, new String[]{}, BYE_CONSTRUCTOR));
```

Comment:
Nitpick: please move the bye command argument ``new String[]{}``(Even though its empty) up to the Argument Group Captures. Then use ``CommandMetadata(BYE_PATTERN, BYE_ARGUMENTS, BYE_CONSTRUCTOR)``

`private static final String[] BYE_ARGUMENTS = {};`